src/main/resources/io/jenkins/plugins/artifact_manager_s3/S3BlobStore/config.jelly

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <j:if test="${!instance.isConfigured()}">
```

Comment:
_Might_ not be what you want, as this will reflect the state as of the last save of `/configure`. So, initially this will still be shown even if the user is filling out that section; and if they blank it out, it will not be shown until they revisit the page. I guess close enough.

Reply:
yep, we know that the Jenkins UI has such kind of limitations, but it is what we have.